browser/ui/toolbar/brave_vpn_menu_model.cc

```diff
@@ -29,6 +34,12 @@ void BraveVPNMenuModel::Build() {
                       IsBraveVPNButtonVisible()
                           ? IDS_BRAVE_VPN_HIDE_VPN_BUTTON_MENU_ITEM
                           : IDS_BRAVE_VPN_SHOW_VPN_BUTTON_MENU_ITEM);
+#if BUILDFLAG(IS_WIN)
+  AddItemWithStringId(IDC_TOGGLE_BRAVE_VPN_TRAY_ICON,
```

Comment:
same. I think this entry also should be shown only when wg feature is enabled?

Reply:
done